python/sglang/srt/layers/sampler.py

```diff
@@ -124,6 +124,7 @@ def forward(
                         sampling_info.top_ps,
                         sampling_info.min_ps,
                         sampling_info.need_min_p_sampling,
+                        sampling_info.sampling_seed,
```

Comment:
We should use hash(sampling_seed, position) as the real seed. Otherwise, each position will use the same seed to sample

Reply:
done